project/Dependencies.scala

```diff
@@ -11,8 +11,8 @@ object Dependencies {
   val ScalaVersion = "2.13.5"
 
   // Note: sync with Protobuf version in Akka gRPC and ScalaPB
-  // https://github.com/akka/akka-grpc/blob/v1.1.1/project/Dependencies.scala#L77
-  val ProtobufVersion = "3.15.1"
+  // https://github.com/akka/akka-grpc/blob/v2.0.0/project/Dependencies.scala#L26
```

Comment:
As we have it from BuildInfo now, we can remove the sync comments.

Reply:
Removed the comments